rskj-core/src/test/java/org/ethereum/jsontestsuite/GitHubBasicTest.java

```diff
@@ -45,20 +44,15 @@
 @Ignore
 public class GitHubBasicTest {
 
+    private static RskSystemProperties config = new RskSystemProperties();
     private static final Logger logger = LoggerFactory.getLogger("TCK-Test");
-    private static final DifficultyCalculator DIFFICULTY_CALCULATOR = new DifficultyCalculator(ConfigHelper.CONFIG);
+    private static final DifficultyCalculator DIFFICULTY_CALCULATOR = new DifficultyCalculator(config);
 
     public String shacommit = "99afe8f5aad7bca5d0f1b1685390a4dea32d73c3";
 
-    @After
-    public void recover() {
-        ConfigHelper.CONFIG.setBlockchainConfig(MainNetConfig.INSTANCE);
-    }
-
     @Test
     public void runDifficultyTest() throws IOException, ParseException {
-
-        ConfigHelper.CONFIG.setBlockchainConfig(MainNetConfig.INSTANCE);
+        config.setBlockchainConfig(new MainNetConfig());
```

Comment:
This is inter-test because you can't forget to do `setBlockchainConfig`. On the other hand, the test is ignored...

Reply:
It won't have trouble on tests since each one assign it's own blockchain config. Either way right this tests aren't being used right now